### PR TITLE
Document the use of config/application.properties

### LINKED ISSUE #4452: Document that one can override runtime properties via config/application.properties
You can override runtime properties in bulk by placing a `config/application.properties` int he working directory of the application.
This is useful for users but not documented.

https://github.com/quarkusio/quarkus/issues/1218#issuecomment-491832844
https://github.com/geoand/quarkus/blob/46dc078de494b331d83d39df8dddcc97e516bf85/core/runtime/src/main/java/io/quarkus/runtime/configuration/ApplicationPropertiesConfigSource.java#L70

No reference in https://quarkus.io/guides/application-configuration-guide

### PR DESCRIPTION
**Pull request by geoand**
Fixes: #4452

### CODE REVIEW

File: docs/src/main/asciidoc/application-configuration-guide.adoc [hunk 1]
```diff
@@ -357,6 +357,10 @@ You can override these runtime properties with the following mechanisms (in decr
 2. using environment variables:
   * for a runner jar: `export QUARKUS_DATASOURCE_PASSWORD=youshallnotpass ; java -jar target/myapp-runner.jar`
   * for a native executable: `export QUARKUS_DATASOURCE_PASSWORD=youshallnotpass ; ./target/myapp-runner`
+3. using a configuration file placed in `$PWD/config/application.properties`
+  * By placing an `application.properties` file inside a directory named `config` which resides in the directory where the application runs, any runtime properties defined
```
[the remark this answers is not in the document]
Comment (geoand) [reply docs/src/main/asciidoc/application-configuration-guide.adoc]: That's interesting! I'll rephrase a little and add it

[the remark this answers is not in the document]
Comment (geoand) [reply docs/src/main/asciidoc/application-configuration-guide.adoc]: Added, thanks for the suggestion!

**Review by gsmet — APPROVED** [review]

**Comment by emmanuelbernard** [comment]
Is there a test testing this so we don’t regress ?

**Comment by geoand** [comment]
I don't think we do. I'll add one shortly.

**Pull request merged**